_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,133 @@
+
+---
+name: video only element transcript
+description: |
+  Non-streaming `video` elements without audio must have all visual information available in a transcript.
+
+success_criterion:
+- 
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video doesn't contain audio.
+
+### Expectation 1
+
+A text transcript is available for the test target, either on the page or available through a link.
+
+### Expectation 2
+
+The text transcript (from expectation 1) contains all the visual information of the test target.
+
+**Note**: A "text transcript" in the context of this rule is defined in WCAG 2 as a [alternative for time based media](https://www.w3.org/TR/WCAG21/#dfn-alternative-for-time-based-media).
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```

Comment:
Please check the wording for this section in other media rules (#204)

Reply:
Done. Thank you.